source/tutorials/importing.rst

```diff
@@ -97,19 +121,94 @@ Importing data
      --input-path emp-paired-end-sequences \
      --output-path emp-paired-end-sequences.qza
 
+Casava 1.8 single-end demultiplexed fastq
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format description
+******************
+
+In the `Casava 1.8 demultiplexed`_ (single-end) format, there is one ``fastq.gz`` file for each sample in the study which contains the `single-end` reads for that sample. The file name includes the sample identifier and should look like ``L2S357_15_L001_R1_001.fastq.gz``. The underscore-separated fields in this file name are:
+
+1. the sample identifier,
+2. the barcode sequence or a barcode identifier,
+3. the lane number,
+4. the direction of the read (i.e. only R1, because these are single-end reads), and
+5. the set number.
+
+Obtaining example data
+``````````````````````
+
+.. download::
+   :url: https://data.qiime2.org/2018.8/tutorials/importing/casava-18-single-end-demultiplexed.zip
+   :saveas: casava-18-single-end-demultiplexed.zip
+
+.. command-block::
+
+   unzip -q casava-18-single-end-demultiplexed.zip
+
+Importing data
+**************
+
+.. command-block::
+
+   qiime tools import \
+     --type 'SampleData[SequencesWithQuality]' \
+     --input-path casava-18-single-end-demultiplexed \
+     --input-format CasavaOneEightSingleLanePerSampleDirFmt \
+     --output-path demux-single-end.qza
+
+Casava 1.8 paired-end demultiplexed fastq
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Format description
+******************
+
+In `Casava 1.8 demultiplexed`_ (paired-end) format, there are two ``fastq.gz`` files for each sample in the study, each containing the forward or reverse reads for that sample. The file name includes the sample identifier. The forward and reverse read file names for a single sample might look like ``L2S357_15_L001_R1_001.fastq.gz`` and ``L2S357_15_L001_R2_001.fastq.gz``, respectively.
+The underscore-separated fields in this file name are:
+
+1. the sample identifier,
+2. the barcode sequence or a barcode identifier,
+3. the lane number,
+4. the direction of the read (i.e. R1 or R2), and
+5. the set number.
+
+Obtaining example data
+``````````````````````
+
+.. download::
+   :url: https://data.qiime2.org/2018.8/tutorials/importing/casava-18-paired-end-demultiplexed.zip
+   :saveas: casava-18-paired-end-demultiplexed.zip
+
+.. command-block::
+
+   unzip -q casava-18-paired-end-demultiplexed.zip
+
+Importing data
+**************
+
+.. command-block::
+   qiime tools import \
+     --type 'SampleData[PairedEndSequencesWithQuality]' \
+     --input-path casava-18-paired-end-demultiplexed \
+     --input-format CasavaOneEightSingleLanePerSampleDirFmt \
+     --output-path demux-paired-end.qza
+
 .. _`manifest file`:
 
 "Fastq manifest" formats
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
+If you don't have either EMP or Casava format, you need to import your data into QIIME 2 manually.
```

Comment:
How about:

```suggestion
If you don't have either EMP or Casava format, you need to import your data into QIIME 2 manually using a "manifest" format.
```
?

Reply:
Sure, though I find the focus on calling "manifest" a format very confusing. Maybe because the "manifest" format has something called a manifest file, but the manifest file itself is not an actual format (it's a file). I can see where this terminology comes from if I try to explain it from a CS point of view, but as a computational biologist it was very confusing to see the same word used for a format and a file.

But keeping things consistent with the rest of the QIIME 2 docs and code is more important than my confusion, so that's just a "for what it's worth!" :)

Reply:
> though I find the focus on calling "manifest" a format very confusing

Oops, sorry, wasn't trying to emphasize that with my suggestion. I guess when I first read it I was thinking, "well, a new user might see this and think they aren't able to use QIIME 2 with their demuxed data," so my suggestion was coming from a place of clarifying that "yep, we can handle that."

I am okay with this being left off (as in, as it was before), I think on second read it is pretty darn clear that this is a statement within the context of the manifest formats section --- I will gladly go with whatever you decide to do.

Reply:
Got it, just made a change that should hopefully address both our concerns! :)

New text:

> If you don't have either EMP or Casava format, you need to import your data into QIIME 2 manually by first creating a "manifest file" and then using the `qiime tools import` command with a few more flags specified than in the EMP or Casava import commands.